tests/TestOne/PackageA/package.json

```diff
@@ -4,39 +4,33 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "PackageB": "../PackageB",
-    "buildtool": "../buildtool"
+    "PackageB": "file:../PackageB",
+    "buildtool": "file:../buildtool",
+    "ocaml": "https://github.com/andreypopp/ocaml.git#esy"
```

Comment:
As per discussion in Discord we decided to leave it as-is for now. Certainly should not depend on patched compiler in real world use cases.